modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/deployable/DeployableProjectModel.java

```diff
@@ -52,11 +53,14 @@ public final class DeployableProjectModel {
   private final Set<BundleDescriptor> sharedLibraries;
   private final Map<BundleDescriptor, List<BundleDependency>> additionalPluginDependencies;
 
+  private final List<Path> resourcesPath;
```

Comment:
isn't this just something intermediate for calculating the value for `resources`?

Reply:
I will need it in the mule-framework to create the classloaders pointing to those resources folders